borsh-rs/borsh/src/ser/mod.rs

```diff
@@ -1,17 +1,20 @@
-use std::borrow::Cow;
-use std::collections::{BTreeMap, HashMap, HashSet};
-use std::convert::TryFrom;
-use std::mem::size_of;
-use std::{io, io::Write};
+use crate::lib::*;
```

Comment:
Would it be too much of a hassle to avoid using `*` imports?

Reply:
fixed